packages/nuxt/src/components/module.ts

```diff
@@ -42,6 +43,12 @@ export default defineNuxtModule<ComponentsOptions>({
         : context.components
     }
 
+    if (nuxt.options.dev || nuxt.options.test) {
```

Comment:
We should probably override component names even in build, so KeepAlive will work consistently in this case.

Reply:
In that case, this would slightly increase the final bundle 🤔 . Do you think this would solve a bug with KeepAlive ?